phantomgames/Search/SearchGameViewController.swift

```diff
@@ -13,82 +13,77 @@ class SearchGameViewController: UIViewController {
     
     @IBOutlet weak private var tableView: UITableView!
     private lazy var viewModel = SearchGameViewModel(repository: SearchGameRepository(), delegate: self)
+    
     // MARK: UI Component
     private let searchController = UISearchController(searchResultsController: nil)
     
+    var gamesURL: String?
+    
     // MARK: Functions
     
     override func viewDidLoad() {
         super.viewDidLoad()
         setupTableView()
         setupSearchController()
-        viewModel.fetchSearchResults()
-    }
+        if let gamesurl = gamesURL {
+            viewModel.fetchSearchResults(fromURL: gamesurl)
+        } else {
+            viewModel.fetchSearchResults()
+        }}
```

Comment:
please put the closing brace of the function on a new line
Not sure why all of these closing braces have been changed. They were correct.
The original issue is that there were no new line breaks between functions.

e.g. func Test {
}
func TestTwo { 
}

should have been 
e.g. func Test {
}
//new line here//
func TestTwo { 
}

These are swiftlint warnings. Why is swiftlint stiilnnot setup for this file?😓

Reply:
swift lint is setup but currently those warnings are not on this file

Reply:
i'm afraid it has not been set up correctly then. It should definitely be picking up these as warnings.😅
as soon as the comments are resolved, i'll approve from my side

Reply:
thank you for the clarification